assets/js/blocks/checkout/input/index.js

```diff
@@ -48,14 +53,36 @@ registerBlockType( 'woocommerce/checkout-input', {
 						) }
 					</Notice>
 				) }
+				{ hasSettings && (
+				<InspectorControls key="inspector">
```

Comment:
This line must be indented one more level, I think:
```suggestion
					<InspectorControls key="inspector">
```